src/umpire/interface/c_fortran/CMakeLists.txt

```diff
@@ -0,0 +1,70 @@
+##############################################################################
+# Copyright (c) 2016-20, Lawrence Livermore National Security, LLC and Umpire
+# project contributors. See the COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (MIT)
+##############################################################################
+set (umpire_interface_c_fortran_headers
+  wrapAllocator.h
+  wrapResourceManager.h
+  wrapUmpire.h
+  umpire.h
```

Comment:
Was there a particular reason you moved this file?

Reply:
Hmm off the top of my head, I'm not sure, I'll have to remind myself! :O

Reply:
Not a problem - it's more consistent this way.